src/vm/eetwain.cpp

```diff
@@ -3251,7 +3267,7 @@ void UnwindEspFrameEpilog(
     _ASSERTE(!info->ebpFrame && !info->doubleAlign);
     _ASSERTE(info->epilogOffs > 0);
     
-    int offset = 0;
+    int offset = SKIP_VZEROUPPER_IF_PRESENT(epilogBase, 0);
```

Comment:
It looks like the code at line 3284 below relies on the fact that the pop ecx is at epilogBase[0] which seems it would not be the case with your change when there can be the vzeroupper at epilogBase[0..2].